docker-compose.yml

```diff
@@ -4,11 +4,13 @@ services:
     image: postgres
     #restart: always
     environment:
+      POSTGRERS_USER: postgres
       POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: travis-ci-test
```

Comment:
We should leave db names as something more generic. Otherwise when someone brings up our docker-compose.yaml on their system they will get a `travis-ci-test` db.